oggm/utils.py

```diff
@@ -218,12 +245,17 @@ def _download_alternate_topo_file(fname):
     """
 
     # temporary check
-    fzipname = fname + '.zip'
+    fzipname = fname.split('.')[0] + '.zip'
     if fzipname not in [
-        'gimpdem_90m.tif.zip',
-        'iceland.tif.zip',
-        'northcanada.tif.zip',
-        'svalbard.tif.zip',
+        'gimpdem_90m.zip',
+        'iceland.zip',
+        'northcanada.zip',
+        'svalbard.zip',
+        'AN1.zip',
+        'AN2.zip',
+        'AN3.zip',
+        'AN4.zip',
+        'AN5.zip'
     ]:
```

Comment:
Maybe we should get rid of this test anyway, because its a maintenance burden (for example, franz joseph is missing and such)